### ISSUE TITLE
STAC JSON Schema format tags use some invalid values

### ISSUE THREAD

**Issue by jason-gilman** [radiantearth/stac-spec#146]
In https://github.com/radiantearth/stac-spec/blob/master/api-spec/WFS3core%2BSTAC.yaml there is a reference to "dateTime" and "url" string formats

```
type: string
format: dateTime
```

Those should be `date-time` and `uri` to be valid JSON schema.

**Comment by m-mohr** [comment]
I agree with the date-time change, but url was a valid format as of the OpenApi 3 spec (anything is allowed, uri is just an example in the spec). URI is actually more permissive than URL and if URI is intended, the respective descriptions should be changed, too

**Comment by m-mohr** [comment]
We actually inherited this from the WFS spec. I opened an issue (https://github.com/opengeospatial/WFS_FES/issues/155) to solve this and then we can update our definitions.
